### ISSUE TITLE
Redis Sentinel Metrics Not Working

### ISSUE THREAD

**Issue by ShineSmile** [bitnami/charts#7419]
**Which chart**:
bitnami/redis
**Describe the bug**
I have deploy Redis with Sentinel and metrics exporter. The service *redis-metrics:9121* works excellent but *redis-sentinel-metrics:9355* not working as expect. This is the segment of redis-sentinel-metrics and I haven't found *sentinalmetrics* port definition in the chart.
``` yaml
spec:
  clusterIP: 10.116.xx.xx
  clusterIPs:
  - 10.116.xx.xx
  ports:
  - name: tcp-metrics
    port: 9355
    protocol: TCP
    targetPort: sentinelmetrics
  selector:
    app.kubernetes.io/instance: redis
    app.kubernetes.io/name: redis
  sessionAffinity: None
  type: ClusterIP
``` 
**Expected behavior**
If sentinel metrics could be provided, please make the service works. If it is obsolete, please delete it.

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.6.1", GitCommit:"61d8e8c4a6f95540c15c6a65f36a6dd0a45e7a2f", GitTreeState:"clean", GoVersion:"go1.16.5"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.4", GitCommit:"3cce4a82b44f032d0cd1a1790e6d2f5a55d20aae", GitTreeState:"clean", BuildDate:"2021-08-11T18:16:05Z", GoVersion:"go1.16.7", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.8-gke.2100", GitCommit:"4cd085fda961821985d176d25b67445c1efb6ba1", GitTreeState:"clean", BuildDate:"2021-07-16T09:22:57Z", GoVersion:"go1.15.13b5", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by alvneiayu** [comment]
hi @ShineSmile 

Please, could you share with me the version that you are using, please?

Sentinel exporter was already removed with [this PR](https://github.com/bitnami/charts/pull/6787)

Thanks a lot

Álvaro

**Comment by ShineSmile** [comment]
> hi @ShineSmile
> 
> Please, could you share with me the version that you are using, please?
> 
> Sentinel exporter was already removed with [this PR](https://github.com/bitnami/charts/pull/6787)
> 
> Thanks a lot
> 
> Álvaro

Thanks with me regrets.
I found that it comes from old release. I have tried clean install with 15.3.1 and no sentinel metrics service any more.

**Comment by alvneiayu** [comment]
hi @ShineSmile 

You're welcome. No problem.

Thanks a lot

Álvaro

**Comment by perlun** [comment]
In case it helps anyone reading this, we have temporarily adopted the Prometheus `redis_sentinel_exporter` project: https://github.com/hiboxsystems/redis_sentinel_exporter/issues/1. We've also published a new 1.7.2 release to the GitHub project.
